price_feed_poller.py

```diff
@@ -157,16 +157,19 @@ def main(args):
     wrbcontract = wrb(w3, config)
     current_block = w3.eth.blockNumber
     print(f"Current block: {current_block}")
-    
-    poll_interval = 60  # seconds
+
+    # Take the poll interval from arguments else make it 60 seconds
+    poll_interval = args.poll_interval if args.poll_interval else 60
+
     # Call main loop
     log_loop(w3, wrbcontract, pricefeedcontracts, account, gas, request_value, poll_interval)
 
-
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(description='Connect to an Ethereum provider.')
     parser.add_argument('--config_file', dest='config_file', action='store', required=True,
                     help='provide the config toml file with the contract and provider details')
+    parser.add_argument('--poll_interval', dest='poll_interval', action='store', type=int, required=False,
```

Comment:
You can specify the default here, I think this way it will show up if you run `--help`.

```suggestion
    parser.add_argument('--poll_internval', dest='poll_interval', action='store', type=int, required=False, default=60
```

Reply:
`internval` is always better than `interval` :+1:

Reply:
fixed haha